src/prefect/infrastructure/docker.py

```diff
@@ -211,6 +211,10 @@ class DockerContainer(Infrastructure):
         default_factory=get_prefect_image_name,
         description="Tag of a Docker image to use. Defaults to the Prefect image.",
     )
+    extra_kwargs: Optional[Dict[str, Any]] = Field(
```

Comment:
I have a preference for `extra_options` as we may use these blocks across multiple languages in the future and kwargs is Python-specific.